securedrop/journalist_templates/edit_account.html

```diff
@@ -1,31 +1,82 @@
     <input name="password" id="password" type="password" />
+
+    {# While non-admin users can only change their own passwords, admin users
+    can make many other changes, which may exclude password changes. #}
+    {% if user %}
+    <em class="light">Leave blank for no change</em>
+    {% endif %}
+
   </p>
   <p>
     <label for="password_again">Change password (confirm)</label>
     <input name="password_again" id="password_again" type="password" />
   </p>
+
+  {# Only admin users may modify the admin status of other users. #}
+  {% if user %}
+  <p>
+    <input name="is_admin" id="is_admin" type="checkbox" {% if user.is_admin %}checked{% endif %} />
+    <label for="is_admin">Is Administrator</label>
+  </p>
+  {% endif %}
+
   <button class="sd-button" type="submit" id="update">UPDATE</button>
 </form>
 
 <hr class="no-line">
 
 <h2>Reset Two Factor Authentication</h2>
+
+{% if user %}
+<p>If a user's two factor authentication credentials have been lost or compromised, you can reset them here. <em>If you do this, make sure the user is present and ready to set up their device with the new two factor credentials. Otherwise, they will be locked out of their account.</em></p>
+{% else %}
 <p>If your two factor authentication credentials have been lost or compromised, or you got a new device, you can reset your credentials here. <em>If you do this, make sure you are ready to set up your new device, otherwise you will be locked out of your account.</em></p>
-<form method="post" action="{{ url_for('account_reset_two_factor_totp') }}" id="reset-two-factor-totp">
+{% endif %}
+
+{% if user %}
+  {% set totp_reset_url = url_for('admin_reset_two_factor_totp') %}
+  {% set hotp_reset_url = url_for('admin_reset_two_factor_hotp') %}
+{% else %}
+  {% set totp_reset_url = url_for('account_reset_two_factor_totp') %}
+  {% set hotp_reset_url = url_for('account_reset_two_factor_hotp') %}
+{% endif %}
+
+<form method="post" action="{{ totp_reset_url }}" id="reset-two-factor-totp">
+{% if user %}
+  <input name="uid" type="hidden" value="{{ user.id }}"/>
+{% endif %}
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}"/>
-  <button class="sd-button" type="submit" class="pull-right">RESET TWO FACTOR AUTHENTICATION (GOOGLE AUTHENTICATOR)</button>
+  <button class="sd-button" type="submit" class="pull-right">RESET TWO FACTOR AUTHENTICATION (APP)</button>
 </form>
 <br />
-<form method="post" action="{{ url_for('account_reset_two_factor_hotp') }}" id="reset-two-factor-hotp">
+<form method="post" action="{{ hotp_reset_url}}" id="reset-two-factor-hotp">
+{% if user %}
+  <input name="uid" type="hidden" value="{{ user.id }}"/>
+{% endif %}
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}"/>
-  <button class="sd-button" type="submit" class="pull-right">RESET TWO FACTOR AUTHENTICATION (HOTP YUBIKEY)</button>
+  <button class="sd-button" type="submit" class="pull-right">RESET TWO FACTOR AUTHENTICATION (HARDWARE TOKEN)</button>
```

Comment:
Can we add some additional explanatory text not on the button indicating that Yubikey is the hardware token and Google Authenticator is the mobile app?